client/src/components/quiz-card.tsx

```diff
@@ -81,21 +81,23 @@ export const QuizCard = ({ quiz, onTagClick, quiz_key }: QuizCardProps) => {
                     </p>
                 </div>
                 <TooltipProvider>
-                    <TooltipAnchor
-                        render={
-                            <span
-                                tabIndex={0}
-                                aria-label={`Rated by ${quiz.rating.count}`}
-                                className={scoreClass({
-                                    score: scoreToInteger(
-                                        quiz.rating.score || 0,
-                                    ),
-                                })}
-                            >
-                                {quiz.rating.score || "N/A"}
-                            </span>
-                        }
-                    ></TooltipAnchor>
+                    {quiz.rating.score && (
```

Comment:
`score` is an integer
if its 0, we will display 0 instead of the component in the paranthesis.

make it 

`quiz.rating.score !== null`